<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Numerical_analysis_lab_02Class</class>
 <widget class="QMainWindow" name="Numerical_analysis_lab_02Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Численные методы: лаба 2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Тестовая задача</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="5">
         <widget class="QChartView" name="test_error_plot"/>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QChartView" name="test_solution_plot"/>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>9999999</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;max|u - v|=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>n</string>
          </property>
         </widget>
        </item>
        <item row="2" column="8">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;at x =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="test_solve_btn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Решить!</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QLabel" name="max_delta_u_v">
          <property name="text">
           <string>-------</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="test_n_input">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1000</string>
          </property>
          <property name="placeholderText">
           <string>1000</string>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="QLabel" name="max_delta_u_v_x_coord">
          <property name="text">
           <string>-------</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6" rowspan="2" colspan="4">
         <widget class="QTableWidget" name="test_table">
          <property name="minimumSize">
           <size>
            <width>455</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>455</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <column>
           <property name="text">
            <string>x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>u</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>v</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>u - v</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Основная задача</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="2">
         <widget class="QPushButton" name="main_solve_btn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Решить!</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;max|v&lt;/span&gt;&lt;span style=&quot; font-size:10pt; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; - v&lt;/span&gt;&lt;span style=&quot; font-size:10pt; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;|=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QChartView" name="main_solution_plot"/>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="max_delta_v_v">
          <property name="text">
           <string>-------</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>441</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;at x = &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="main_n_input">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1000</string>
          </property>
          <property name="placeholderText">
           <string>1000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QChartView" name="main_error_plot"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>n</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QLabel" name="max_delta_v_v_x_coord">
          <property name="text">
           <string>-------</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" rowspan="2" colspan="4">
         <widget class="QTableWidget" name="main_table">
          <property name="minimumSize">
           <size>
            <width>455</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>455</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <column>
           <property name="text">
            <string>x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>v_1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>v_2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>v_1 - v_2</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>test_n_input</tabstop>
  <tabstop>test_solve_btn</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>test_error_plot</tabstop>
  <tabstop>test_solution_plot</tabstop>
 </tabstops>
 <resources>
  <include location="numerical_analysis_lab_02.qrc"/>
 </resources>
 <connections/>
</ui>
